beacon-chain/blockchain/head.go

```diff
@@ -93,12 +90,16 @@ func (s *Service) updateHead(ctx context.Context, balances []uint64) ([32]byte,
 		if err != nil {
 			return [32]byte{}, err
 		}
+		st, err := s.cfg.StateGen.StateByRoot(ctx, s.ensureRootNotZeros(headStartRoot))
```

Comment:
The call to `ensureRootNotZeroes` is redundant.

Reply:
Nice catch thanks